Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -458,14 +494,21 @@ public class EOSSessionsManager : IEOSSubManager
 
         /// <summary>
         /// After a successful <see cref="RefreshSession(string)"/> and <see cref="OnRefreshSessionFindSessionsCompleteCallback(ref SessionSearchFindCallbackInfo)"/> callback call,
-        /// this is run to inform UI of the updated Session information.
+        /// this is run to inform UI of the updated Session information. This should be subscribed to by your program's UI.
```

Comment:
A bit redundant. Did you intend to replace the comment instead of amending it? Either way, capitalization.

Reply:
I don't see the capitalization fault, though I do see a "you/your" pronoun usage to replace.

I want a clear way to communicate, HEY USER! Subscribe to this or you're missing out!